lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -83,7 +83,10 @@ $proctoring-banner-text-size: 14px;
 
     .exam-text {
       display: inline-block;
-      width: calc(100% - 250px);
+
+      @media(min-width: 768px) {
```

Comment:
Nit: I think this should be written as `@include media-breakpoint-up(md) {` for consistency/bootstrapiness.

Reply:
Works well. Thanks!